namer/marathon/src/main/scala/io/buoyant/namer/marathon/MarathonInitializer.scala

```diff
@@ -64,7 +66,17 @@ case class MarathonConfig(
     case None => 80
   }
   private[this] def getUriPrefix = uriPrefix.getOrElse("")
-  private[this] def getTtl = ttlMs.getOrElse(5000).millis
+
+  @JsonIgnore
+  private[this] def getJitteredTtl(ttl: Int, jitterConf: Int): Stream[Duration] = {
```

Comment:
feel free to make this package private if you want to be able to test it (`private[marathon]`)